### DISCUSSION TITLE
not registered migration

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#3530] [On-Premise: Installation & Updates] [answered]
When updating Kimai to ver. 1.24.0, I get the following message:
```
Kimai installation running ...
==============================

Database `kimai2` for connection named default already exists. Skipped.

 [WARNING] You have 1 previously executed migrations in the database that are   
           not registered migrations.                                           

 >> 2022-06-14 14:42:27 (DoctrineMigrations\Version20211022105517)

 [OK] Already at the latest version ("DoctrineMigrations\Version20220722125847")
```
Can I ignore it or should I take actions to register this dubiously unregistered migration (if so, how)?

**Comment by kevinpapst** [comment, marked as answer]
I don't know that migration file. Did you use dev branches at some point? You can safely ignore that message.

Or even better (but be careful) you can remove that migration from the database:

From the table `migration_versions` find the matching line and delete it:
> 2022-06-14 14:42:27 (DoctrineMigrations\Version20211022105517)

**Reply by BeckeBauer** [reply]
Thanks for the hint - I switched to dev when I experimented with the synology AD login (probably some left over). I cleaned the DB as suggested and the warning is gone - thanks for your quick feedback.
